integration/inspektor-gadget/audit_seccomp_test.go

```diff
@@ -100,13 +137,18 @@ spec:
       - name: sourceprofile
         configMap:
           name: myseccompprofile
-EOF
-`, ns),
-			ExpectedRegexp: fmt.Sprintf("daemonset.apps/seccomp-copier created"),
+`, ns)
+
+	commands := []*Command{
+		CreateTestNamespaceCommand(ns),
+		{
+			Name:           "CreateSeccompProfile",
+			Cmd:            fmt.Sprintf("echo '%s' | kubectl apply -f -", seccompInstallerYaml),
```

Comment:
I prefer here documents with EOF. But here, `echo '%s'` works because `seccompInstallerYaml` does not contain any `'`.

Reply:
Switched to EOF. Thanks for the suggestion!